### PR TITLE
*: fix license header

### LINKED ISSUE #27356: Tracking issue for handling illegal license header
This is a follow up to #26956

Files below with an illegal license header, should be evaluated carefully.

* [x] 'cmd/explaintest/config.toml'
* [x] 'cmd/importer/config.toml'
* [x] 'cmd/portgenerator/portgenerator.go'
* [x] 'ddl/ddl.go'
* [x] 'ddl/ddl_api.go'
* [x] 'ddl/mock.go'
* [x] 'ddl/options.go'
* [x] 'executor/aggfuncs/func_stddevpop_test.go'
* [x] 'executor/aggfuncs/func_stddevsamp_test.go'
* [x] 'executor/aggfuncs/func_varpop_test.go'
* [x] 'executor/aggfuncs/func_varsamp_test.go'
* [x] 'executor/index_lookup_merge_join_test.go'
* [x] 'executor/pkg_test.go'
* [x] 'expression/aggregation/base_func_test.go'
* [x] 'expression/aggregation/util_test.go'
* [x] 'expression/builtin.go'
* [x] 'expression/builtin_info.go'
* [x] 'expression/builtin_math.go'
* [x] 'expression/builtin_miscellaneous.go'
* [x] 'expression/builtin_miscellaneous_test.go'
* [x] 'expression/builtin_string.go'
* [x] 'expression/builtin_time.go'
* [ ] 'metrics/alertmanager/tidb.rules.yml'
* [ ] 'metrics/topsql.go'
* [ ] 'planner/core/cache_test.go'
* [ ] 'planner/core/logical_plans_test.go'
* [ ] 'planner/core/partition_prune.go'
* [ ] 'planner/core/partition_pruning_test.go'
* [ ] 'planner/core/rule_aggregation_push_down.go'
* [ ] 'planner/core/rule_max_min_eliminate.go'
* [ ] 'planner/core/rule_partition_processor.go'
* [ ] 'planner/core/rule_predicate_push_down.go'
* [ ] 'planner/core/telemetry.go'
* [ ] 'plugin/conn_ip_example/manifest.toml'
* [ ] 'server/conn.go'
* [ ] 'server/conn_stmt.go'
* [ ] 'server/packetio.go'
* [ ] 'server/server.go'
* [ ] 'server/util.go'
* [ ] 'session/bootstrap.go'
* [ ] 'session/session.go'
* [ ] 'session/tidb.go'
* [ ] 'store/driver/kv_test.go'
* [ ] 'store/mockstore/unistore/config/config-template.toml'
* [ ] 'store/mockstore/unistore/server/server.go'
* [ ] 'table/column.go'
* [ ] 'table/table.go'
* [ ] 'table/tables/tables.go'
* [ ] 'tests/globalkilltest/config.toml'
* [ ] 'tests/readonlytest/readonly_test.go'
* [ ] 'tools/check/check-timeout.go'
* [ ] 'tools/check/revive.toml'
* [ ] 'types/convert.go'
* [ ] 'types/etc.go'
* [ ] 'types/fuzzMarshalJSON.go'
* [ ] 'types/fuzzNewBitLiteral.go'
* [ ] 'types/fuzzNewHexLiteral.go'
* [ ] 'util/collate/collate_bench_test.go'
* [ ] 'util/format/format.go'
* [ ] 'util/logutil/slow_query_logger.go'
* [ ] 'util/mvmap/fnv.go'
* [ ] 'util/prefix_helper.go'
* [ ] 'util/resourcegrouptag/resource_group_tag.go'
* [ ] 'util/set/set_with_memory_usage_test.go'
* [ ] 'util/testkit/fake.go'
* [ ] 'util/testleak/leaktest.go'
* [ ] 'util/timeutil/time_test.go'
* [ ] 'util/topsql/reporter/mock/server.go'

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

### What problem does this PR solve?

Issue Number: close #27356 

### Release note

```release-note
None
```

### CODE REVIEW

File: table/table.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/pingcap/tidb/util/dbterror"
 )
 
-// Type , the type of table, store data in different ways.
+// Type the type of table, store data in different ways.
```
Comment (disksing) [inline table/table.go:34]:
```suggestion
// Type is used to distinguish between different tables that store data in different ways.
```

Comment (tisonkun) [reply table/table.go]: updated.

**Review by disksing — APPROVED** [review]

**Review by zhouqiang-cl — APPROVED** [review]
LGTM

**Comment by DavidKorczynski** [comment]
I noticed that the OSS-Fuzz build has not been working since this PR. This is because this PR removes the 3 files containing fuzzers - is this intentional, i.e. do you not want your project fuzzed?

**Pull request merged**